src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -76,7 +76,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setPerson(personToEdit, editedPerson);
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
-        if (editPersonDescriptor.getName().isPresent()) { // if name has been edited
+        if (editedPerson.getName().equals(personToEdit.getName())) { // if name has been edited
```

Comment:
♻️ Refactor proposed: Could this be a separate line of code, in the spirit of [making code self-explanatory](https://nus-cs2103-ay2526s1.github.io/website/se-book-adapted/chapters/codeQuality.html#introduction-6)?

```suggestion
        boolean hasNameBeenEdited = editedPerson.getName().equals(personToEdit.getName());
        if (hasNameBeenEdited) {
```